App.js

```diff
@@ -16,9 +16,9 @@ const sentryApi = secrets[namespace].sentryApi;
 if (sentryApi?.dsn) {
   Sentry.init({
     dsn: sentryApi.dsn,
-    enableNative: true, // NOTE: Native crash reporting is not available with the classic build system (expo build:[ios|android]), but is available via EAS Build.
+    enableNative: true // NOTE: Native crash reporting is not available with the classic build system (expo build:[ios|android]), but is available via EAS Build.
```

Comment:
from reading https://docs.sentry.io/platforms/react-native/configuration/options/#enableNative it seams, that this is `true` by default now?

Reply:
checking the repo it also looks like this: https://github.com/getsentry/sentry-react-native/blob/30e2d760ae502b1131a88084c27814033efc89f0/src/js/wrapper.ts#L198

Reply:
Yes. Looks true by default. Then we can also remove this value. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/6b10abf5ed81c30531429b499b33373f3f4fa22c